fireant/database/column.py

```diff
@@ -0,0 +1,103 @@
+from pypika.queries import Column as PypikaColumn
+
+
+class Column:
```

Comment:
Why not just use PypikaColumn directly? What does this class add?

Reply:
This fireant Column class is an abstraction of database specific columns and used as intermediate entity for translations between specific columns. It's able but not limited to generate a PypikaColumn. The PypikaColumn represents the SQL-representation of a column in a query for a given name and type.